00-introduction.md

```diff
@@ -21,6 +21,7 @@ This is version 0.
 7. [BOLT #7](07-routing-gossip.md): P2P Node and Channel Discovery
 8. [BOLT #8](08-transport.md): Encrypted and Authenticated Transport
 9. [BOLT #9](09-features.md): Assigned Feature Flags
+11. [BOLT #11](11-payment-encoding.md): Invoice Protocol for Lightning Payments
```

Comment:
So just because Lightning proposes things built on top of Bitcoin, it therefore is a part of Bitcoin, but BU's proposed consensus changes to Bitcoin itself don't get to count?  What about coinbase account-to-account transfers or coinbase-account-to-bitstamp-account transfers, can we get a BIP for those?  What about the miner fast relay network that is already in use today, why wouldn't that have a BIP?

If it isn't related directly to Bitcoin core / consensus, it doesn't need a BIP.  The only fundamental difference here is that @luke-jr is attempt to have BOLT's and lightning appear to carry the same weight that are currently associated with BIP's.  This inadvertently waters down the meaning of what a BIP is and what qualifies, and turns it into a subjective discussion of what gets to be called a BIP and what doesn't.  There's no technical benefit for the two to be combined, and it doesn't help Bitcoin either.